### ISSUE TITLE
Lack of Organiser Notification Emails

### ISSUE THREAD

**Issue by sam596** [thewca/worldcubeassociation.org#3011]
Just as an idea, less of an issue as such. I feel like Organisers should be notified when one of these occur:

- They've been added as an organiser to a competition.
- Their competition has been submitted to the Board for review.
- Their competition has been announced.

I have never had any of these notifications for a competition I have organised, in fact the first time I realised one of my competitions had been announced was a post on Facebook.

Is there any reason this can't/shouldn't be implemented?

**Comment by jfly** [comment]
> They've been added as an organiser to a competition.

This seems fine to me!

> Their competition has been submitted to the Board for review.

This seems potentially problematic to me, as CC-ing the organizer on these email threads may not be appropriate, as the Board and the Delegate(s) may be discussing how capable the organizer(s) are. It may be best to create a brand new email for this.

The safest option would be to create a separate email for this

> Their competition has been announced.

Similar to the "submitted to the Board for review" email, it may not be appropriate to CC the organizer on this email, so if we implement this, it may be best to create a brand new email for this.

**Comment by sam596** [comment]
>  as CC-ing the organizer on these email threads may not be appropriate,

would Bcc work for the initial (i'm assuming automated) email? That way the organiser wouldn't be replied to on a reply all. Forgive me if i'm missing something.

**Comment by sam596** [comment]
Unfortunately my Ruby/Rails knowledge is very lacking, so no, I wouldn't know where to start to be honest.

**Comment by LinusFresz** [comment]
can take a look at that after I am done with the registration mails. :-)

**Comment by jfly** [comment]
Wooohoo, thanks @LinusFresz! When you do get around to this, please start by sending an email in plain English explaining what we plan to do to the Board. As discussed here, I think there are some subtleties such that we'd like to make sure they're ok with these changes before we go to the effort to implement them.
